package.json

```diff
@@ -18,12 +16,12 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/opbeat/babel-plugin-add-react-displayname.git"
+    "url": "git+https://github.com/sergei-startsev/babel-plugin-react-displayname-path.git"
   },
   "author": "Ron Cohen",
```

Comment:
and maybe do something with author? https://docs.npmjs.com/files/package.json#people-fields-author-contributors